File: Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []
    
    def __init__(self,license, drivers, typeCarAccepted, seatsMaterial):
        super().__init__(license,drivers)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: PHP/uberVan.php
<?php
require_once('car.php');
class UberVan extends Car {
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, $drivers,$typeCarAccepted, $seatsMaterial){
        parent::__construct($license, $drivers);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
    
}